src/test/transaction_tests.cpp

```diff
@@ -470,12 +470,14 @@ BOOST_AUTO_TEST_CASE(test_big_witness_transaction) {
 
     // check all inputs concurrently, with the cache
     PrecomputedTransactionData txdata(tx);
-    boost::thread_group threadGroup;
+    std::vector<std::thread> threadGroup;
```

Comment:
same comment; this is fixed in my WIP lockfree stuff so may want to leave it assuming I get mine merged.